code/modules/admin/verbs/mentorhelp.dm

```diff
@@ -0,0 +1,561 @@
+/client/var/mentorhelptimerid = 0	//a timer id for returning the mhelp verb
+/client/var/datum/mentor_help/mentor_current_ticket	//the current ticket the (usually) not-mentor client is dealing with
+
+//
+//TICKET MANAGER
+//
+
+GLOBAL_DATUM_INIT(mhelp_tickets, /datum/mentor_help_tickets, new)
+
+/datum/mentor_help_tickets
```

Comment:
After thinking about it a lot, tickets should all share a common parent rather than being a bunch of duplicated code like this. Do something like `/datum/tickets` holding most of the code and specific code can be in `/datum/tickets/admin` and `/datum/tickets/mentor`